LANGUAGE.md

```diff
@@ -11,7 +11,7 @@ Should dogescript be ported to other languages, the `js` portion may be changed
 
 * dogescript uses single quotes for strings. Double quotes are not supported.
 * dogescript uses 4 space characters for indentation. Tabs are not supported.
-* dogescript seperates statements by newlines by default. In true-doge mode, they are separated by 3 spaces.
+* dogescript separates statements by newlines by default. In true-doge mode, they are separated by 3 spaces.
```

Comment:
This change seems irrelevant?

Reply:
I noticed it's misspelled, I can revert it